llama_index/indices/list/retrievers.py

```diff
@@ -201,6 +201,52 @@ def _retrieve(self, query_bundle: QueryBundle) -> List[NodeWithScore]:
         return results
 
 
+class SummaryIndexKeyWordsRetriever(BaseRetriever):
```

Comment:
Hmm, this isn't really keyword search. More like full-text search? Thoughts?

Reply:
It filters nodes based on keywords in the query. The query should be in this format: 'keyword1, keyword2, ...' The separator used when splitting the query could be modified with the parameter 'sep'.